docs/docs/api/LayoutAnimations/KeyframeAnimations.md

```diff
@@ -0,0 +1,179 @@
+---
+id: keyframeAnimations
+title: Keyframe Animations
+sidebar_label: Keyframe Animations
+---
+
+The document explains how you can define complex animation using simple and popular animation definitions schema - Keyframes.
+
+## How to define Keyframe animation?
+
+### 1. Import Keyframe
+
+```js
+import { Keyframe } from 'react-native-reanimated';
+```
+
+### 2. Create Keyframe object, define initial and final state
+
+In Keyframe's constructor pass object with definitions of your animation. Object keys correspond to animation progress,
+so to 0 assign the style, you want for your object at the beginning of the animation and to 100 assign the style you want for your object to have at the end of the animation.
+
+```js
+import { Keyframe } from 'react-native-reanimated';
+
+const keyframe = new Keyframe({
+    0: {
+      transform: [ { rotate: '0deg' }],
+    },
+    100: {
+      transform: [ { rotate: '45deg' }],
```

Comment:
🌵 Unnecessary whitespace in transform array.

Reply:
Actually in most of the examples whitespace is inconsistent. Could you fix that, please?